lib/miq_pglogical.rb

```diff
@@ -143,6 +143,21 @@ def self.default_excludes
     YAML.load_file(Rails.root.join("config/default_replication_exclude_tables.yml"))[:exclude_tables] | ALWAYS_EXCLUDED_TABLES
   end
 
+  def self.save_replication_type(type)
```

Comment:
I don't think it's worth defining a new method for this. You can go ahead and just use `MiqRegion.replication_type=` directly in the other two methods.